tox.ini

```diff
@@ -92,10 +71,14 @@ commands = /usr/bin/bash -c './scripts/migrations-check.sh'
 allowlist_externals = bash
 commands = /usr/bin/bash -c './scripts/schema-check.sh'
 
-[isort]
-profile = black
-skip = migrations,venv,.tox,src,attic
+[testenv:ruff-check]
+deps = ruff==0.12.3
+commands = ruff check --extend-ignore I osidb collectors apps
```

Comment:
We could make use of [substituting values from different section ](https://tox.wiki/en/3.24.0/config.html#substitution-for-values-from-other-sections) to have the `ruff` version defined only in one place.